src/Model/Document/Editable/Outputchanneltable.php

```diff
@@ -276,37 +280,34 @@ public function load()
         $this->setElements();
     }
 
-    /**
-     * Methods for Iterator
-     */
-    public function rewind()
+    public function rewind(): void
     {
         $this->setElements();
         reset($this->elements);
     }
 
-    public function current()
+    public function current(): mixed
     {
         $this->setElements();
 
         return current($this->elements);
     }
 
-    public function key()
+    public function key(): mixed
     {
         $this->setElements();
 
         return key($this->elements);
     }
 
-    public function next()
+    public function next(): void
     {
         $this->setElements();
 
-        return next($this->elements);
+        next($this->elements);
```

Comment:
This change would be quite inconsistent with the other iterator methods, i would say that this change was fine, it's `rewind()` that wasn't returning

Reply:
Sorry, double-checked and this change was actually fine

The Iterator "wants" `void` for `next` and `rewind`
![image](https://user-images.githubusercontent.com/6014195/203112582-d959e202-dcf0-4cae-b47f-ed445b6b9f40.png)